### ISSUE TITLE
Add metrics for Command and Control to HTTP adapter

### ISSUE THREAD

**Issue by sysexcontrol** [eclipse/hono#632]
We need Grafana dashboards (scoped to tenant) for:

- command delivered to device before ttd expires
- ttd expired with no command received from applications
- command response delivered from device to application

**Comment by kaniyan** [comment]
I made changes to collect below metrics for Command & Control and store them in Influx-db. 

- Commands delivered (command delivered to device before ttd expires)
- TTD expiries (ttd expired with no command received from applications)
- Commad responses delivered (command response delivered from device to application)

Find below the screenshots of Dashboard with Command and Control metrics. Its just an idea, how command and  control metrics can be showed in the dashboard.  @ctron @pellmann @sophokles73 @sysexcontrol any opinion on this?  :)  

![honodashboard-1](https://user-images.githubusercontent.com/18659982/43309931-75e9003a-9186-11e8-8a3a-8d833a8cb5d3.jpg)
From #711 , I understood that the graphs at the top show the no. of messages per tenant (aggregated across all adapter types). In this case, it shows the no. of commands (delivered to devices) per tenant (aggregated across all adapter types).

![honodashboard-2](https://user-images.githubusercontent.com/18659982/43309971-8dccfac6-9186-11e8-9015-3f957a379ac3.jpg)

![honodashboard-3](https://user-images.githubusercontent.com/18659982/43309974-8ff9c32e-9186-11e8-9844-be6236cf6c34.jpg)

**Comment by pellmann** [comment]
I like the boards! Have you already included the queries for MQTT?

**Comment by kaniyan** [comment]
Thank you for the feedback. Queries for MQTT are already included in this Dashboard.

**Comment by sophokles73** [comment]
@kaniyan are you also going to add these metrics to the MQTT adapter? If so, we could re-use this issue and simply change its title accordingly.

**Comment by kaniyan** [comment]
@sophokles73  yes, I will continue with metrics for MQTT adapter. We can re-use this issue.
